python/catkin/test_results.py

```diff
@@ -134,6 +134,15 @@ def test_results(test_results_dir):
                 print('Skipping "%s": %s' % (name, str(e)))
                 continue
             results[name] = (num_tests, num_errors, num_failures)
+            if show_detail and (num_errors + num_failures > 0) :
+                print("Test %s was failed" % (name))
+                print("-------------------------------------------------")
+                detail=open(filename_abs,"rb").read()
+                if sys.version_info[0] >= 3:
+                    detail=detail.decode('utf-8')
+                detail=detail.replace('\n',"\n").replace('&gt;',">").replace('&lt;',"<")
```

Comment:
I assume you want to replace "\\n" with "\n"? And not the same string with itself as it is currently?